.github/workflows/npm.yml

```diff
@@ -0,0 +1,51 @@
+name: NPM
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "contracts/**"
+      - "package.json"
+      - "yarn.lock"
+  pull_request: #TODO: remove before merging
+  workflow_dispatch:
+
+jobs:
+  npm-compile-publish-contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          registry-url: "https://registry.npmjs.org"
+          cache: "yarn"
+
+      - name: Resolve latest contracts
+        run: |
+          yarn add \
+            @keep-network/keep-core \
+            @keep-network/tbtc
+
+      - name: Compile contracts
+        run: yarn deploy --network hardhat --write true
+
+      - name: Copy artifacts
+        run: |
+          mkdir -p artifacts
+          cp -r build/contracts/* artifacts/
+
```

Comment:
We don't need this step in a setup where we use hardhat.
The copying we need will be performed by `npm publish` job with execution of `prepublishOnly` script.

For reference of the package structure please see https://github.com/keep-network/coverage-pools/issues/159.
```suggestion
```

Reply:
Right, I actually thought I removed this one, but apparently not. Fixed in 738b247b5c158e61a77fe697eebca48991e70fcd.